add-ons/che/README.md

```diff
@@ -25,33 +26,49 @@ The best way of using this add-on is via the [`minishift add-ons apply`](https:/
 
 <a name="install-add-on"></a>
 ### Install add-on
+Che default stacks require root privilege to run sshd. Without root privilege, workspaces will take more time to timeout and start.
+One workaround would be to enable the `anyuid` addon. Another way would be to [replace the existing stacks](#replace-stacks) with the ones without the `sshd` agent.
```

Comment:
Does `anyuid` work? Someone has tested if sshd get started? Because we explicitely start the container as unprivileged in the [OpenShiftConnector](https://github.com/eclipse/che/blob/master/plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java#L1489).

Reply:
@sunix - we also can add one more template parameter called `MAVEN_MIRROR_URL` , we have just added made the che stack images to use MAVEN_MIRRORS like how other openshift images does

Reply:
@l0rd I've tested with `ayuid` and startup is faster for sure

Reply:
@kameshsampath not sure to get this, but maybe this should be part of another PR ?

Reply:
@l0rd , is this related to it https://github.com/minishift/minishift-addons/pull/43#issuecomment-337445709

Reply:
ok thanks @budhrg and @sunix